.github/workflows/build.yml

```diff
@@ -10,9 +10,8 @@ on:  # https://docs.github.com/en/actions/reference/workflows-and-actions/events
     # https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#schedule
     - cron: '0 15 1 * *'
   push:
-    branches-ignore:  # build all branches except:
-    - 'dependabot/**'     # prevent GHA triggered twice (once for commit to the branch and once for opening/syncing the PR)
-    - 'dependencies/pdm'  # prevent GHA triggered twice (once for commit to the branch and once for opening/syncing the PR)
+    branches:
+    - main
```

Comment:
@1cu I should have seen this earlier but this is not correct. We do not only build the main branch. Push on any branch should trigger a build, especially the release branch but also in forks on the branches people are working on.